charts/opensearch-cluster/README.md

```diff
@@ -39,7 +39,7 @@ The following table lists the configurable parameters of the Helm chart.
 | `cluster.general.snapshotRepositories` | list | `[]` | Opensearch snapshot repositories configuration |
 | `cluster.general.vendor` | string | `"Opensearch"` |  |
 | `cluster.general.version` | string | `"2.3.0"` | Opensearch version |
-| `cluster.bootstrap.additionalConfig` | object | `{}` | bootstrap additional configuration, key-value pairs that will be added to the opensearch.yml configuration |
+| `cluster.bootstrap.env` | list | `[]` | bootstrap pod env variables |
```

Comment:
Unclear - `additionalConfig` is removed here but still seems to be supported in the crd? please explain. We do need the ability to add additional configs to `opensearch.yml`

Reply:
> Do we need to allow different config per node pool?

Yes